client/my-sites/checkout/composite-checkout/components/contact-details-container.tsx

```diff
@@ -52,13 +52,12 @@ export default function ContactDetailsContainer( {
 		.filter( ( product ) => isDomainProduct( product ) || isDomainTransfer( product ) )
 		.filter( ( product ) => ! isDomainMapping( product ) )
 		.map( getDomain );
-	const { updateDomainContactFields, updateCountryCode, updatePostalCode, updateEmail } =
+	const { updateDomainContactFields, updateSomeFields, updateEmail } =
```

Comment:
This absolutely makes sense to do, I was just wondering if it might be better to name this as something slightly more descriptive like `updateSomeTaxFields`? Throughout, not just here.

Reply:
Good idea.